Process.c

```diff
@@ -786,9 +786,10 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
       bool lastItem = (this->indent < 0);
       int indent = (this->indent < 0 ? -this->indent : this->indent);
 
-      for (int i = 0; i < 32; i++) {
+      for (int i = 31; i >= 0; i--) {
          if (indent & (1U << i)) {
             maxIndent = i + 1;
```

Comment:
How about removing this loop together with `maxIndent` and replacing the next `for` loop with something the following?
```
while (indent > 1) {
   if (indent & 1) { ... } else { ... }
   indent >>= 1;
}
```

Reply:
Is this equivalent to the following?
```c
if (indent == 0) {
   maxIndent = 0;
} else {
   maxIndent = __builtin_clz(indent) + 1;
} 
```

Reply:
Is this equivalent to the following?
```c
if (indent == 0) {
   maxIndent = 0;
} else {
   maxIndent = (32 - __builtin_clz(indent));
} 
```

Reply:
IIUC, `__builtin_clz` is a GCC builtin, but `htop` does not require `gcc`. This could be an optimization, but does not sound worth it to me as we iterate over the same bit positions below in any case and perform string formatting, which is heavy enough to hide additional costs.

Reply:
> How about removing this loop together with `maxIndent` and replacing the next `for` loop with something the following?
> 
> ```
> while (indent > 1) {
>    if (indent & 1) { ... } else { ... }
>    ...
>    indent >>= 1;
> }
> ```

indeed, thanks!